.travis.yml

```diff
@@ -46,6 +46,10 @@ after_script:
 
 jobs:
   include:
+      - name: "Check Code Style"
```

Comment:
This will cause a separate build, which is time consuming. I guess verification of code style could be done together with verification of code quality (Clippy).